docs/support/tutorials/gis/eo_guide.md

```diff
@@ -0,0 +1,257 @@
+# Earth Observation guide
+
+This guide aims to help users to work with Earth Observation (EO) data using CSC's computing resources. The purpose of this guide is to help you find the right data and tools for your raster data based EO tasks. If you are interested in the fundamentals of EO, please check the resources at the bottom of the page.
+
+**What are the benefits of using EO data**
+
+* Possibility to observe wide area at same time
+* Non-intrusive
+* Same sensor for different parts of the world
+* Time series 
+
+!!! default "Raster data format"
+
+    * EO data often several bands, stored one file per band or several bands in one file (multiband files).
+    * Grid of pixel values
+    * Example of continuous data
+    * Georeference: coordinate for the top left pixel , the size of each pixel in the X direction, the size of each pixel in the Y direction, and the amount (if any) by which the product is rotated.
```

Comment:
```suggestion
    * EO data often several bands, stored one file per band or several bands in one file (multiband files).
    * Grid of pixel values
    * Example of continuous data
    * Georeference: coordinate for the top left pixel , the size of each pixel in the X direction, the size of each pixel in the Y direction, and the amount (if any) by which the product is rotated.
```
EO data is usually available in [raster format]( 
https://towardsdatascience.com/the-ultimate-beginners-guide-to-geospatial-raster-data-feb7673f6db0). The most common file formats are [GeoTiff](https://en.wikipedia.org/wiki/GeoTIFF) and [GeoJPEG2000](http://giswiki.org/wiki/GeoJPEG2000).